### PR TITLE
tuesday content fixes

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: pegasus/cache/i18n/en-US.yml [hunk 1]
```diff
@@ -281,7 +281,7 @@
   inspirational_videos_more: "More <a href=\"https://www.youtube.com/playlist?list=PLzdnOPI1iJNfpD8i4Sx7U0y2MccnrNZuP\">inspirational videos</a> by role models and celebrities"
   playlab_choose_theme: "Choose your Play Lab theme"
   playlab_classic_title: "Classic"
-  playlab_gumball_title: "Gumball"
+  playlab_gumball_title: "The Amazing World of Gumball"
```
Comment (breville) [inline pegasus/cache/i18n/en-US.yml:284]: are we changing this in the .script and in the scripts.en.yml as well?

File: pegasus/sites.v3/code.org/views/plc_csp_nav.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#about_nav
+  %span.about_list
+    -current_url = '/educate/plc/csp'
+    -{'Apply to be a Participant'=>'csp-application',  'About the Curriculum'=>current_url+'#curriculum', 'About the Professional Learning Program'=>current_url+'#about', 'How does it work?'=> current_url+ '#components', 'What are the commitments?'=> current_url+'#commitments', 'Who can participate?'=> current_url+'#participate', 'Where is it offered?'=>current_url+'#locations'}.each_pair do |label,url|
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/plc_csp_nav.haml:4]: i like to break these up into multi-lines if possible

**Comment by breville** [comment]
the huge images should be resized by pegasus before serving using fit- style URLs, but apart from that lgtm.

File: pegasus/cache/i18n/en-US.yml (hunk 1 above)
Comment (tanyaparker) [inline pegasus/cache/i18n/en-US.yml:284]: @caleybrock for a later PR -- we should make sure it's the full name everywhere. /s/gumball has a different title.

**Pull request merged**